core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -249,9 +250,18 @@ public static BuiltInType fromString(String value) {
 
     /**
      * Vineflower Decompiler configuration
+     *
+     * @Deprecated use {@code quarkus.package.decompiler} instead
```

Comment:
Maybe add a note after which version it will be removed?

Reply:
I don't know when it will be removed :)

Reply:
For sure, but I usually add a hint like "will be removed at some point after Quarkus 3.10" or something like that. To define the "transition period". So that the next one who reads the javadoc can actually remove it. Because otherwise it may stay there forever...